tests/strict_mock_testslide.py

```diff
@@ -509,6 +510,40 @@ def works_with_wraps(self):
                                         )
                                         self.assertEqual(method("hello"), "mock: hello")
 
+                                    @context.example
+                                    def works_with_Mock_parameter(self):
+                                        class Dummy_param:
+                                            def get_string():
+                                                return "Dummy_param: not mocked"
+
+                                        class Dummy:
+                                            def method(self, param: Dummy_param):
+                                                return f"Dummy: {param.get_string()}"
+
+                                        my_strict_mock = StrictMock(template=Dummy)
```

Comment:
Shouldn't this be:
```
my_strict_mock = Mock(spec=Dummy)
```
as per test title?

Reply:
What I'm validating is that a strict mock, that has to validate a call to a mocked method, validates correctly when one of the params passed to that call is actually a mock with a spec